include/ipc-client.h

```diff
@@ -24,6 +24,10 @@ char *get_socketpath(void);
  */
 int ipc_open_socket(const char *socket_path);
 /**
+ * Issues a single IPC command without reading response.
+ */
+void ipc_single_command_no_response(int socketfd, uint32_t type, const char *payload, uint32_t *len);
```

Comment:
I'm not sure how to throttle on sway side. Since all requests processed synchronous, we cannot limit request count (without counting requests in last _n_ seconds), but we could make socket non-blocking, which will help with locks, but on high request rates there may be unexpected problems with data loss.